third_party/validator/iam_helpers_test.go

```diff
@@ -0,0 +1,148 @@
+package google
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestMergeBindings(t *testing.T) {
```

Comment:
Can you add a test for merging multiple roles in addition to merging members?

Reply:
Will do

Reply:
Added a big kahuna case